lib/improver/tests/utilities/test_cube_manipulation.py

```diff
@@ -527,6 +545,694 @@ def test_cubelist_different_var_names(self):
+                                cubelist[1].attributes["grid_id"])
+
+    def test_cubelist_grid_id_in_list(self):
+        """Test that the utility updates grid_id if in list and not matching"""
+
+        cube1 = self.cube_ukv.copy()
+        cube2 = self.cube.copy()
+        cube1.attributes.update({'grid_id': 'ukvx_standard_v1'})
+        cube2.attributes.update({'grid_id': 'enukx_standard_v1'})
+
+        cubelist = iris.cube.CubeList([cube1, cube2])
+
+        equalise_cube_attributes(cubelist)
+
+        self.assertAlmostEquals(cubelist[0].attributes["grid_id"],
+                                cubelist[1].attributes["grid_id"])
+        self.assertAlmostEquals(cubelist[0].attributes["grid_id"],
+                                'ukx_standard_v1')
+
+    def test_cubelist_grid_id_in_list2(self):
+        """Test that the utility updates grid_id if in list and not matching
+        where grid_id has already been updated to ukv_standard_v1"""
+
+        cube1 = self.cube_ukv.copy()
+        cube2 = self.cube.copy()
+        cube1.attributes.update({'grid_id': 'ukvx_standard_v1'})
+        cube2.attributes.update({'grid_id': 'ukx_standard_v1'})
+
+        cubelist = iris.cube.CubeList([cube1, cube2])
+
+        equalise_cube_attributes(cubelist)
+
+        self.assertAlmostEquals(cubelist[0].attributes["grid_id"],
+                                cubelist[1].attributes["grid_id"])
+        self.assertAlmostEquals(cubelist[0].attributes["grid_id"],
+                                'ukx_standard_v1')
+
+    def test_cubelist_grid_id_not_in_list(self):
+        """Test leaves grid_id alone if grid_id not matching and not in list
+        In this case the cubes would not merge.
+        """
+
+        cube1 = self.cube_ukv.copy()
+        cube2 = self.cube.copy()
+        cube1.attributes.update({'grid_id': 'ukx_standard_v1'})
+        cube2.attributes.update({'grid_id': 'unknown_grid'})
+
+        cubelist = iris.cube.CubeList([cube1, cube2])
+
+        equalise_cube_attributes(cubelist)
+
+        self.assertIn("grid_id", cubelist[0].attributes.keys())
+        self.assertEqual(cubelist[0].attributes["grid_id"],
+                         'ukx_standard_v1')
+        self.assertIn("grid_id", cubelist[1].attributes.keys())
+        self.assertEqual(cubelist[1].attributes["grid_id"],
+                         'unknown_grid')
+
+    def test_cubelist_title_identical(self):
+        """Test that the utility does nothing to title if they match"""
+
+        cube1 = self.cube_ukv.copy()
+        cube2 = self.cube.copy()
+        cube1.attributes.update({'title':
+                                 'Operational UKV Model Forecast'})
+        cube2.attributes.update({'title':
+                                 'Operational UKV Model Forecast'})
+
+        cubelist = iris.cube.CubeList([cube1, cube2])
+
+        equalise_cube_attributes(cubelist)
+        self.assertAlmostEquals(cubelist[0].attributes["title"],
```

Comment:
Why AlmostEquals for a string comparison?

Reply:
Because when I first wrote these I didn't know there was an assertEqual in unittest. I thought I got rid of these but obviously left a couple